frontend/src/routes/ProviderConnections/ProviderConnections/ProviderConnections.test.tsx

```diff
@@ -147,6 +184,23 @@ describe('provider connections page', () => {
         await clickByText('common:cancel')
         await waitForNotText('common:cancel')
     })
+
+    test('If cloud.redhat.com providerconnection and no discoveryconfig configured, show action available', async () => {
+        render(<TestProviderConnectionsPage providerConnections={[cloudRedHatProviderConnection]} />)
+        await waitForText(cloudRedHatProviderConnection.metadata!.name!)
+        await waitForText('connections.actions.enableClusterDiscovery')
+    })
+
+    test('If cloud.redhat.com providerconnection and discoveryconfig configured, do not show action available', async () => {
+        render(
+            <TestProviderConnectionsPage
+                providerConnections={[cloudRedHatProviderConnection]}
+                discoveryConfigs={[discoveryConfig]}
+            />
+        )
+        await waitForText(cloudRedHatProviderConnection.metadata!.name!)
+        await !waitForText('connections.actions.enableClusterDiscovery')
```

Comment:
i'm not sure this is doing what we think it is?
I think this should be
```
await waitForNotText('connections.actions.enableClusterDiscovery')
```

Reply:
I wanted to test specifically that this text did not show up in the UI, this is what differed it from the above test on line 191.